docs/cel2/l2-operator-guide.md

```diff
@@ -368,16 +368,15 @@ make geth
 OP_GETH_IMAGE=us-west1-docker.pkg.dev/devopsre/celo-blockchain-public/op-geth:celo8
 
 # Copying the jwt secret from the op-node directory
-cp ${OP_NODE_DIR}/jwt.txt ${OP_GETH_DIR}/jwt.txt
+cp ${OP_NODE_DIR}/jwt.txt ${OP_GETH_DATADIR}/jwt.txt
 
 docker run -d \
   --name op-geth \
   --network=host \
-  -v ${OP_GETH_DIR}:/celo \
-  -e GETH_DATADIR=/celo \
+  -v ${OP_GETH_DATADIR}:/datadir \
```

Comment:
@jcortejoso can you verify if this is a legitimate change?

Reply:
`GETH_DATADIR` helps as you don't need to carry over the `--datadir` flag for any other command. Especially convenient for `geth attach`

Reply:
Ahh ok got you, hadn't thought about that.